### PR TITLE
DFG-Checklisten-Implementierung

### PR DESCRIPTION
**Pull request by Zack-83**
This PR offers the support for the DFG checklist through addition of 
- a DFG-Checklists question catalog 
- a DFG-Checklists guide for personalising
- 15 option sets 
- [coming soon] a DFG-Checklists view 

| Change        | Key                                  | Contents   |                                                                                                                                                                       | to do |
| ------------- | ------------------------------------ | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----- |
| New catalog   | dfg-checklist.xml                    |            |                                                                                                                                                                       |       |
| New optionset | dfg_dataset_size_options             | 3 options  | What is the anticipated data volume?                                                                                                                                  |       |
| New optionset | dfg_archive                          | 3 options  | Are you planning to archive your data in a suitable infrastructure? How and where?                                                                                    |       |
| New optionset | dfg_description_standards            | 7 options  | What approaches are being taken to describe the data in a comprehensible manner (such as the use of existing standards for metadata or documentation, or ontologies)? |       |
| New optionset | dfg_new_data                         | 15 options | How does your project generate new data?                                                                                                                              |       |
| New optionset | dfg_publication-restrictions         | 5 options  | Do you anticipate any implications or restrictions regarding subsequent publication or accessibility?                                                                 |       |
| New optionset | dfg_resources                        | 19 options | Which resources (costs, time or other) are required to implement adequate handling of research data within the project?                                               |       |
| New optionset | dfg_reuse                            | 6 options  | Are existing data reused?                                                                                                                                             |       |
| New optionset | dfg_reuse_context                    | 5 options  | Are these data suitable for subsequent use in other contexts?                                                                                                         |       |
| New optionset | dfg_scientific_guidelines            | 4 options  | Are there any significant research codes or professional standards to be taken into account?                                                                          |       |
| New optionset | dfg_storage                          | 9 options  | How are data stored and secured during the project duration?                                                                                                          |       |
| New optionset | dfg_using_when                       | 9 options  | When are the research data available for use by third parties?                                                                                                        |       |
| New optionset | dfg_dataset_quality                  | 8 options  | What measures are being adopted to ensure high data quality?                                                                                                          |       |
| New optionset | dfg_dataset_format                   | 29 options | Which data types in terms of data formats (e.g. image data, text data or measurement data) arise in your project?                                                     |       |
| New optionset | dfg_preservation_motivation_options  | 8 options  | Which criteria are used to select research data to make them available for subsequent use by others?                                                                  |       |
| New optionset | control_tools_long                   | 18 options | Are quality controls foreseen and if so, how do they operate?                                                                                                         |       |

### CODE REVIEW

File: rdmorganiser/options/rdmo.xml [hunk 1]
```diff
@@ -1542,6 +1542,17 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Zack-83) [inline rdmorganiser/options/rdmo.xml:1555]:
```suggestion
```

File: rdmorganiser/questions/DFG-Checkliste.xml [hunk 2]
```diff
@@ -0,0 +1,1204 @@
...
+		<widget_type>text</widget_type>
+		<value_type>text</value_type>
+		<maximum/>
+		<minimum/>
+		<step/>
+		<unit/>
+		<width/>
+		<optionsets/>
+		<conditions/>
+	</question>
```
Comment (Zack-83) [inline rdmorganiser/questions/DFG-Checkliste.xml:1053]:
```suggestion
```

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
Translation work is not done. The English version is not in sync with the German.

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
It looks good from the high level perspective. This is without going into the details of every text in the catalog ect. 

I suggest one more change before merging.
 
*  In the README.md the Checklist catalog is listed to be located under shared catalogs, but they are located in the 'rdmorganiser' directory. I suggest that we keep it in place and change the README.md accordingly.

**Comment by Zack-83** [comment]
Prefixes replaced. Import works.

Possible additional changes (let's talk about, @jmfrenzel ):

* remove `dfg_` where not strictly necessary;
* harmonise optionset keys (some end with `_options`, others do not);
* reconciliate the competing optionsets:
  * `dataset_size_options` and `dfg_dataset_size_options`
  * `preservation_motivation_options` and `dfg_preservation_motivation_options`
  * `control_tools` and `control_tools_long`

**Review by jmfrenzel — APPROVED** [review]
looks good to me

**Pull request merged**